i18n/hu.json

```
@@ -0,0 +1,65 @@
+{
+	"@metadata": {
+		"authors": [
+			"Hungarian Scratch translators", "ahmetlii"
+		]
+	},
+	"skinname-scratchwikiskin2": "ScratchWikiHéj",
+	"scratchwikiskin-desc": "Egy héj a Scratch Wiki számára.",
+	"scratchwikiskin-create": "Alkoss",
+	"scratchwikiskin-create-project": "Készítsen egy projektet",
+	"scratchwikiskin-create-page": "Hozzon létre egy projektet",
+	"scratchwikiskin-create-page-url": "Projekt:Hozzon létre egy projektet",
```

"Project" is a namespace name and should not be translated.
```suggestion
	"scratchwikiskin-create-page-url": "Project:Hozzon létre egy projektet",
```